easybuild/easyconfigs/c/CrystFEL/CrystFEL-0.11.1-foss-2024a.eb

```
@@ -0,0 +1,46 @@
+easyblock = 'CMakeMake'
+
+name = 'CrystFEL'
+version = '0.11.1'
+
+homepage = 'http://www.desy.de/~twhite/crystfel/'
+description = """CrystFEL is a suite of programs for processing diffraction data acquired "serially" in a "snapshot"
+manner, such as when using the technique of Serial Femtosecond Crystallography (SFX) with a free-electron laser
+source."""
+
+toolchain = {'name': 'foss', 'version': '2024a'}
+
+source_urls = ['http://www.desy.de/~twhite/crystfel/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['6139b818079a16aa4da90344d4f413810e741c321013a1d6980c01f5d79c7b3a']
```

checksum on website (and currently downloadable) is
```suggestion
checksums = ['bd9b8df5d627a30c920bf397d04641d266e1a013b24623967d7f5deac97594a4']
```
see: https://www.desy.de/~twhite/crystfel/download.html

---

`6139b818079a16aa4da90344d4f413810e741c321013a1d6980c01f5d79c7b3a` is for 0.8.0. It is in https://github.com/easybuilders/easybuild-easyconfigs/pull/8407/files#diff-d707294fba1a9aafa4a04db501674ec9048463edd6ca2f29953654004faa29f4R15